tests/integration/parallel_skip.json

```diff
@@ -1,187 +1,35 @@
-  "test_storage_kafka/test.py::test_row_based_formats",
-  "test_storage_kafka/test.py::test_block_based_formats_1",
-  "test_storage_kafka/test.py::test_block_based_formats_2",
-  "test_storage_kafka/test.py::test_system_kafka_consumers",
-  "test_storage_kafka/test.py::test_system_kafka_consumers_rebalance",
-  "test_storage_kafka/test.py::test_system_kafka_consumers_rebalance_mv",
-  "test_storage_kafka/test.py::test_formats_errors",
-  "test_storage_kafka/test.py::test_multiple_read_in_materialized_views",
-  "test_storage_kafka/test.py::test_kafka_null_message",
-
-  "test_storage_kafka/test_produce_http_interface.py::test_kafka_produce_http_interface_row_based_format",
-
-  "test_storage_kerberized_kafka/test.py::test_kafka_json_as_string",
-  "test_storage_kerberized_kafka/test.py::test_kafka_json_as_string_request_new_ticket_after_expiration",
-  "test_storage_kerberized_kafka/test.py::test_kafka_json_as_string_no_kdc",
-  "test_storage_kerberized_kafka/test.py::test_kafka_config_from_sql_named_collection",
-
-  "test_dns_cache/test.py::test_ip_change_drop_dns_cache",
-  "test_dns_cache/test.py::test_ip_change_update_dns_cache",
-  "test_dns_cache/test.py::test_dns_cache_update",
-  "test_dns_cache/test.py::test_user_access_ip_change",
-  "test_dns_cache/test.py::test_host_is_drop_from_cache_after_consecutive_failures",
-  "test_dns_cache/test.py::test_dns_resolver_filter",
-
-  "test_https_replication/test_change_ip.py::test_replication_when_node_ip_changed",
-
-  "test_host_regexp_multiple_ptr_records/test.py::test_host_regexp_multiple_ptr_v4_fails_with_wrong_resolution",
-  "test_host_regexp_multiple_ptr_records/test.py::test_host_regexp_multiple_ptr_v4",
-  "test_host_regexp_multiple_ptr_records/test.py::test_host_regexp_multiple_ptr_v6",
-
-  "test_grpc_protocol/test.py"
+  "test_atomic_drop_table/",
+  "test_attach_without_fetching/",
+  "test_broken_part_during_merge/",
+  "test_cleanup_dir_after_bad_zk_conn/",
+  "test_consistent_parts_after_clone_replica/",
+  "test_cross_replication/",
+  "test_ddl_worker_non_leader/",
+  "test_delayed_replica_failover/",
+  "test_dictionary_allow_read_expired_keys/",
+  "test_disabled_mysql_server/",
+  "test_distributed_ddl",
+  "test_distributed_respect_user_timeouts/",
+  "test_drop_replica",
+  "test_hedged_requests/",
+  "test_hedged_requests_parallel/",
+  "test_https_replication/",
+  "test_insert_into_distributed",
+  "test_keeper_multinode_simple/",
+  "test_limited_replicated_fetches/",
+  "test_materialized_mysql_database/",
+  "test_parts_delete_zookeeper/",
+  "test_polymorphic_parts/",
+  "test_quorum_inserts_parallel/",
+  "test_random_inserts/",
+  "test_reload_clusters_config/",
+  "test_replace_partition/",
+  "test_replicated_database",
+  "test_replicated_fetches_timeouts/",
+  "test_storage_kafka/",
+  "test_storage_kerberized_kafka/",
+  "test_system_clusters_actual_information/",
+  "test_system_metrics/",
+  "test_system_replicated_fetches/"
```

Comment:
:+1:

Reply:
But you skipped some tests, was that the intention?

Reply:
Interesting, many things like `distributed_ddl` aren't filtered... what a mess